public/components/getting_started/components/utils.tsx

```diff
@@ -20,16 +39,55 @@ const fetchAssets = async (tutorialId: string, assetFilter?: 'dashboards' | 'ind
   return responeData;
 };
 
-export const UploadAssets = async (tutorialId: string, mdsId: string, mdsLabel: string) => {
+export const UploadAssets = async (
+  tutorialId: string,
+  mdsId: string,
+  mdsLabel: string,
+  schema: ICollectorSchema[],
+  collectorOptions: Array<EuiSelectableOption<CollectorOption>>
+) => {
   const { setToast } = useToast();
   const http = coreRefs.http;
 
+  let selectedIntegration: string | undefined;
+  const checkedCollector = collectorOptions.find((collector) => {
+    return !!collector.checked;
+  });
+
+  if (checkedCollector !== undefined) {
+    if (checkedCollector.value === 'otel') {
```

Comment:
My PR adjust the otel value to be broken into the three different paths 'otelLogs' , 'otelMetrics' and 'otelTraces' which will all have slightly different templates to push.